src/PublicApiAnalyzers/UnitTests/DeclarePublicApiAnalyzerTests.cs

```diff
@@ -1224,6 +1224,51 @@ public struct Nested { }
                 );
         }
 
+        [Fact]
+        public async Task ShippedTextWithMissingImplicitStructConstructorAsync()
```

Comment:
[cosmetic] We usually don't use the `Async` suffix for tests

```suggestion
        public async Task ShippedTextWithMissingImplicitStructConstructor()
```